docs/tutorials/task.cis-to-component-definition/transformation.md

```diff
@@ -0,0 +1,1092 @@
+# Tutorial: Setup for and use of ComplianceAsCode profile to OSCAL Component Definition transformer
+
+Here are step by step instructions for setup and transformation of [ComplianceAsCode](https://github.com/ComplianceAsCode/content) profile data files into [NIST](https://www.nist.gov/) standard [OSCAL](https://pages.nist.gov/OSCAL/) [Component Definition](https://pages.nist.gov/OSCAL/reference/latest/component-definition/json-outline/) using the [compliance-trestle](https://ibm.github.io/compliance-trestle/) tool.
+
+## *Objective*
+
+How to transform one or more `.profile` compliance files into a standardized `OSCAL.json` file.
+
+There are 2 short steps shown below.
+The first is a one-time check/set-up of your environment.
+The second is a one-command transformation from `.profile` to `OSCAL.json`.
+
+## *Step 1: Environment setup*
+
+Linux, Mac
+
+<details>
+<summary>Windows</summary>
+
+Make these changes:
+
+<ul>
+<li>use backslashes `\` for file paths
+<li>use `md` instead of mkdir -p
+<li>put the url in double quotes for `curl`
+<li>use `more` instead of cat
+</ul>
+</details>
+
+- Insure you have a modern [Python](https://www.python.org/downloads/) (3.7, 3.8, 3.9).
+
+```
+$ python -V
+Python 3.8.7
+```
+
+- Setup a virtual environment.
+
+```
+$ cd
+$ python -m venv venv.trestle
+$ source venv.trestle/bin/activate
+(venv.trestle)$
+
+```
+
+- Insure you have a modern [pip](https://pip.pypa.io/en/stable/installation/) (19.x or greater).
+
+```
+(venv.trestle)$ pip --version
+pip 21.2.4 from /home...
+```
+
+- Install [compliance-trestle](https://ibm.github.io/compliance-trestle/).
+
+```
+(venv.trestle)$ pip install compliance-trestle
+Looking in indexes: https://pypi.org/simple,...
+
+```
+
+- Check trestle viability (and view help).
+
+```
+(venv.trestle)$ trestle -h
+usage: trestle [-h] 
+{add,assemble,author,create,describe,href,import,init,merge,partial-object-validate,remove,replicate,split,task,validate,version} ...
+
+```
+
+- Create trestle workspace.
+
+```
+(venv.trestle)$ mkdir trestle.workspace
+(venv.trestle)$ cd trestle.workspace
+(venv.trestle)$ trestle init
+Initialized trestle project successfully in /home/<user>/trestle.workspace
+```
+
+- View configuration information.
+
+```
+(venv.trestle)$ trestle task cis-to-component-definition -i
+trestle.core.commands.task:99 WARNING: Config file was not configured with the appropriate section for the task: "[task.cis-to-component-definition]"
+Help information for cis-to-component-definition task.
+
```

Comment:
You have a warning message here.  Can you just delete it, or do you want users to be aware this warning will appear?

Reply:
This is what you get when no `-c` is specified. It might be nice if this warning was not issued when `-i` is specified. I'm happy to open an issue if you like. I think no change is req'd to this tutorial since this reflects the current reality.